docs/docs/rest-api/public/api/v2/types/group.raml

```diff
@@ -100,3 +65,16 @@ types:
         minimum: 0
       version?:
         type: datetime
+      enforceRole?:
+        type: EnforceRole
+  GroupPartialUpdate:
+    type: object
+    properties:
+      id: strings.PathId
+      enforceRole?:
+        type: EnforceRole
+  EnforceRole:
+      type: string
+      enum:
+        - Off
+        - Top
```

Comment:
I know this is as spec'ed but it reads weird to me. `Off` sounds like the other value would be `On`, which would be a boolean. IIRC this was coming from _we might allow more than one level later on_, would would go from `Off, Top` to … `Top-2`? `Two`?

I don't know, I think we should test / have tested with users whether they understand what `enforceRole: Top` means.

Reply:
This is according to the spec for `--default_enforce_group_role`. I don't really mind but thought that `Top` allows `Top-2` as you mentioned. @timcharper, what are your thoughts?

Reply:
The defaulting behavior is an enum to allow specification of different depths of enforce role. At a group level it should just be a boolean

Reply:
Sorry. I don't get it. What does `--default_enforce_group_role` describe?

Also, if it is a boolean what happens when we have
```yaml
group-1:
  enforceRole: true,
  group-2:
    enforceRole: false
    group-3:
      enforceRole: true
```

Reply:
From my understanding, `default_enforce_group_role` defines the default for the enforceGroupRole-Boolean flag on groups. As we only allow top-level-groups to have that flag, the value for `default_enforce_group_role` can only be "Off" (no groups have that flag enabled) or "Top" (top-level groups have that flag enabled). If we were to allow more levels later on, we might add "All" or some other value.

Regarding the nesting of enforceRole-Flags: I'd assume that disabling the enforceRole-Flag shouldn't be possible. I.e. if a group at one level defines enforceRole to true, no sub-group can set the flag to false again.

Reply:
Ok, I think we have to rename `--default_enforce_group_role` to `--group_role_behaviour` or something similar. The names and boolean vs enum is quite confusing.